code/datums/jobs.dm

```diff
@@ -138,6 +138,47 @@
 				M.choose_name(3, src.name, default)
 				if(M.real_name != default && M.real_name != orig_real)
 					phrase_log.log_phrase("name-[ckey(src.name)]", M.real_name, no_duplicates=TRUE)
+				if((M.job == "Clown") || (M.job == "Mime"))
+					var/make_bracelet = FALSE //friendship bracelets!!!
+					for(var/mob/MOB in mobs)
```

Comment:
this should either break after finding the first match or at least have a cap

if an admin sets the clown job limit high as a gimmick each person gets a bracelet for every other person. That's a lot of bracelets. May also apply to mass respawning people as clowns and mimes.

Reply:
i was thinking about that :/

Reply:
i set the limit to 4 sets of bracelets that can generate.